functions.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+
+/**
+ * функция, которая обрезает текстовое содержимое если оно превышает заданное число символов
+ * @param string $description текстовое содержимое
+ * @param string $append добавляется  в конец этой строки знак многоточия
+ * @param string $read_more добавляется к итоговой строке тег с ссылкой «Читать далее»
+ * @param integer $length заданное число символов
+ * @return string Возвращает обрезанное  текстовое содержимое
+ */
+
+function truncate(
+    string $description,
+    string $append = "&hellip;",
+    string $read_more = "<a class='post-text__more-link' href='#'>Читать далее</a>",
```

Comment:
В шаблонах допускается переменные в формате snake_case, но в php функциях, классах и т.д. только camelCase. 
```suggestion
    string $readMore = "<a class='post-text__more-link' href='#'>Читать далее</a>",
```

Reply:
Использовать такой аргумент в данном ключе не целесообразно. Если бы у тебя было 2 и более мест, где нужно использовать эту функцию и при этом менять формат ссылки "читать далее", то может быть, но сейчас это избыточно.